ocamltest/main.ml

```diff
@@ -196,7 +196,7 @@ let find_test_dirs dir =
     Array.iter (fun s ->
         if ignored s then ()
         else begin
-          let s = Filename.concat dir s in
+          let s = dir ^ "/" ^ s in
```

Comment:
This change looks fishy to me. Do we understand why it is necessary to make Cygwin happy?

Reply:
If we use `Filename.concat` then the output will have things that look like `dir\file` on Windows and when we pass that to the (Cygwin) shell (from inside the `Makefile`) that causes trouble.

I agree this is a bit fishy and am not too proud of it. Suggestions welcome.

Reply:
I'm happy to go with "fishy but works" if it does indeed work. I will run a `precheck` to ensure that it works on our test configurations.

Reply:
Precheck job: https://ci.inria.fr/ocaml/job/precheck/305/

Reply:
Re-launched the precheck job after the latest changes: https://ci.inria.fr/ocaml/job/precheck/306/

Reply:
Precheck passed, except for cygwin64 which failed but am not sure it is related...